infrastructure/metadata/infrastructure/server/context.any.js.ini

```diff
@@ -1,6 +1,6 @@
 [context.any.worker-module.html]
   expected:
-    if product == "firefox": TIMEOUT # https://bugzilla.mozilla.org/show_bug.cgi?id=1247687
+    if product == "firefox": [TIMEOUT, OK] # https://bugzilla.mozilla.org/show_bug.cgi?id=1247687
```

Comment:
Seems like this shouldn't be intermittent, but should just be expected to pass now, given the bug is fixed.

Reply:
Updated metadata to reflect that.

Why we did not detect this earlier? Is this because the fix is at browser side, so this test is not run until now?